### PR TITLE
update "runtime" docs in zh version

### PR DESCRIPTION
**Pull request by SherlockShemol**
* **Please check if the PR fulfills these requirements**
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

docs update 

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by wbc6080** [comment]
It looks like the DCO signature is still unresolved. You need to execute `git rebase HEAD~6 --signoff` on your local branch, then `git log` check the commit record to see if all your commits are signed. After adding them all, then execute the `git push --force-with-lease origin master` command, which should solve the problem.

**Comment by SherlockShemol** [comment]
> It looks like the DCO signature is still unresolved. You need to execute `git rebase HEAD~6 --signoff` on your local branch, then `git log` check the commit record to see if all your commits are signed. After adding them all, then execute the `git push --force-with-lease origin master` command, which should solve the problem.

yes I have followed the instruction and signiture does appears in my git log.But DCO just doesn't pass.sorry for my unskilled pr.give me some time and I will figure out.

**Comment by SherlockShemol** [comment]
> It looks like the DCO signature is still unresolved. You need to execute `git rebase HEAD~6 --signoff` on your local branch, then `git log` check the commit record to see if all your commits are signed. After adding them all, then execute the `git push --force-with-lease origin master` command, which should solve the problem.看起来 DCO 签名问题仍未解决。您需要在本地分支上执行 `git rebase HEAD~6 --signoff` ，然后 `git log` 检查提交记录，看看是否所有提交都已签名。全部添加完毕后，再执行 `git push --force-with-lease origin master` 命令，问题应该就解决了。

Finally I found out the problem.sorry for so many ugly commit.

**Comment by Shelley-BaoYue** [comment]
You can use git reset to merge these commits into one. The following steps are for your reference. 
-  git reset bfc529d045f0407e770972489ede033ebcc1fa37  （Remember do not use git reset --hard ）
- git add * 
- git commit --amend -s 
- git push -f origin master

**Comment by SherlockShemol** [comment]
> You can use git reset to merge these commits into one. The following steps are for your reference.
> 
>     * git reset [bfc529d](https://github.com/kubeedge/website/commit/bfc529d045f0407e770972489ede033ebcc1fa37)  （Remember do not use git reset --hard ）
> 
>     * git add *
> 
>     * git commit --amend -s
> 
>     * git push -f origin master

Thanks and it works! Time to learn more git command.

**Comment by Shelley-BaoYue** [comment]
I found that soma images have also been updated in this PR, but I don’t quite understand what has been updated. Could you please describe the update?

**Review by Shelley-BaoYue — COMMENTED** [review]
I found that some pictures have been updated in this PR, but I don’t quite understand what has been updated. Could you please describe it?

**Comment by SherlockShemol** [comment]
> I found that some pictures have been updated in this PR, but I don’t quite understand what has been updated. Could you please describe it?

yes it's weird.I have done nothing to those images.I fork and clone the repo and go directly to modify the "runtime" file.But those images is shown to be modified. Maybe it's because of [mode of the files](https://stackoverflow.com/questions/5787937/git-status-shows-files-as-changed-even-though-contents-are-the-same).

**Comment by Shelley-BaoYue** [comment]
> > I found that some pictures have been updated in this PR, but I don’t quite understand what has been updated. Could you please describe it?
> 
> yes it's weird.I have done nothing to those images.I fork and clone the repo and go directly to modify the "runtime" file.But those images is shown to be modified. Maybe it's because of [mode of the files](https://stackoverflow.com/questions/5787937/git-status-shows-files-as-changed-even-though-contents-are-the-same).

How about do not select these image files when `git add`?

**Comment by SherlockShemol** [comment]
> > > I found that some pictures have been updated in this PR, but I don’t quite understand what has been updated. Could you please describe it?
> > 
> > 
> > yes it's weird.I have done nothing to those images.I fork and clone the repo and go directly to modify the "runtime" file.But those images is shown to be modified. Maybe it's because of [mode of the files](https://stackoverflow.com/questions/5787937/git-status-shows-files-as-changed-even-though-contents-are-the-same).
> 
> How about do not select these image files when `git add`?

yes, give me some time.

**Comment by SherlockShemol** [comment]
> > > I found that some pictures have been updated in this PR, but I don’t quite understand what has been updated. Could you please describe it?
> > 
> > 
> > yes it's weird.I have done nothing to those images.I fork and clone the repo and go directly to modify the "runtime" file.But those images is shown to be modified. Maybe it's because of [mode of the files](https://stackoverflow.com/questions/5787937/git-status-shows-files-as-changed-even-though-contents-are-the-same).
> 
> How about do not select these image files when `git add`?

Done.Thank you.

File: i18n/zh/docusaurus-plugin-content-docs/current/setup/prerequisites/runtime.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 3
 ---
 ## containerd
 
-Docker 18.09 and up ship with `containerd`, so you should not need to install it manually. If you do not have `containerd`, you may install it by running the following:
+Docker 18.09 及以上版本已随附`containerd`，因此无需手动安装。如果没有`containerd`，可以通过运行以下命令来安装：
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/setup/prerequisites/runtime.md:7]: 随附这个词有些奇怪，或许可以换为
```suggestion
Docker 18.09 及以上版本已附带`containerd`，因此无需手动安装。如果没有`containerd`，可以通过运行以下命令来安装：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/setup/prerequisites/runtime.md [hunk 2]
```diff
@@ -18,15 +18,15 @@ containerd config default > /etc/containerd/config.toml
 systemctl restart containerd
 ```
 
-When using `containerd` shipped with Docker, the cri plugin is disabled by default. You will need to update `containerd`’s configuration to enable KubeEdge to use `containerd` as its runtime:
+在使用随 Docker 装运的`containerd`时，cri 插件默认为禁用。您需要更新 containerd 的配置，以使 KubeEdge 使用`containerd`作为其运行时：
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/setup/prerequisites/runtime.md:21]: 同上
```suggestion
在使用 Docker 附带的`containerd`时，cri 插件默认为禁用。您需要更新 containerd 的配置，以使 KubeEdge 使用`containerd`作为其运行时：
```

**Pull request closed without merging**